src/clusterfuzz/_internal/bot/tasks/utasks/fuzz_task.py

```diff
@@ -1967,12 +1967,14 @@ def _make_session(uworker_input):
   )
 
 
-HANDLED_ERRORS = [
-    uworker_msg_pb2.ErrorType.UNHANDLED,
-    uworker_msg_pb2.ErrorType.FUZZ_BUILD_SETUP_FAILURE,
-    uworker_msg_pb2.ErrorType.FUZZ_DATA_BUNDLE_SETUP_FAILURE,
-    uworker_msg_pb2.ErrorType.FUZZ_NO_FUZZER,
-]
+_ERROR_HANDLER = uworker_handle_errors.CompositeErrorHandler({
+    uworker_msg_pb2.ErrorType.FUZZ_NO_FUZZER:
+        handle_fuzz_no_fuzzer,
+    uworker_msg_pb2.ErrorType.FUZZ_BUILD_SETUP_FAILURE:
+        handle_fuzz_build_setup_failure,
+    uworker_msg_pb2.ErrorType.FUZZ_DATA_BUNDLE_SETUP_FAILURE:
+        handle_fuzz_data_bundle_setup_failure,
+}).compose_with(uworker_handle_errors.UNHANDLED_ERROR_HANDLER)
```

Comment:
Do you think UNHANDLED should be special cases so this doesn't need to be repeated everywhere?

Reply:
It's not used in `regression_task.py`. Should it?

In any case, if all error handlers should do this by default, then happy to add this to the `CompositeErrorHandler` class. In the interest of landing this quickly, I can do that in a follow-up PR?